style.css

```diff
@@ -1,8 +1,169 @@
-#app {
-  font-family: Avenir, Helvetica, Arial, sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@500&family=Rubik:wght@600&display=swap');
+
+
+:root {
+  --primary-color: #026a90;
+  --secondary-color: lightgray;
+}
+
+* {
+  box-sizing: border-box;
+
+}
+
+body {
+  background-color: #FFFFFF;
+  font-family: sans-serif;
+  margin: 0;
+}
+
+header{
+  margin: 0 auto;
+  width: 100%;
   text-align: center;
-  color: #2c3e50;
-  margin-top: 60px;
 }
+h1{
+  margin: 0 0 50px;
+  font-size: 48px;
+  font-weight: 400;
+
+
+font-family: 'Roboto Mono', monospace;
```

Comment:
Use formatter like prettier to have consistent spacing and indents throughout the code.